engine/src/test/java/io/camunda/zeebe/engine/state/compensation/CompensationSubscriptionStateTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.state.compensation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.engine.state.mutable.MutableCompensationSubscriptionState;
+import io.camunda.zeebe.engine.state.mutable.MutableProcessingState;
+import io.camunda.zeebe.engine.util.ProcessingStateExtension;
+import io.camunda.zeebe.protocol.impl.record.value.compensation.CompensationSubscriptionRecord;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ProcessingStateExtension.class)
+public class CompensationSubscriptionStateTest {
+
+  private static final String TENANT_ID = "tenantId";
+  private static final long PROCESS_INSTANCE_KEY = 1L;
+  private static final long PROCESS_DEFINITION_KEY = 1L;
+  private static final String COMPENSABLE_ACTIVITY_ID = "compensableActivityId";
+  private static final long COMPENSABLE_ACTIVITY_SCOPE_ID = 1L;
+  private static final String THROW_EVENT_ID = "throwEventId";
+  private static final long THROW_EVENT_INSTANCE_KEY = 1L;
```

Comment:
:wrench: We should use different numbers for different properties. Otherwise, a failure might be not detected because all properties have the same value (i.e. `1L`).